blogs/schema/model.py

```diff
@@ -0,0 +1,37 @@
+from typing import List, Optional
+from pydantic import BaseModel
+
+class Blogs(BaseModel):
+    id: Optional[int]= None
+    title: str
+    body: str
+
+class Show_user(BaseModel):
```

Comment:
```suggestion
class ShowUser(BaseModel):
```
follow a consistent naming convention.